README.md

```diff
@@ -1 +1,25 @@
-# moodle-tool_advancedreplace
\ No newline at end of file
+# moodle-tool_advancedreplace
+
+This is a Moodle plugin that allows administrators to search and replace strings in the Moodle database.
+
+Administrators can search and replace strings in tables and columns of the Moodle database.
+They can use simple text search or regular expressions.
+
+## GDPR
+The plugin does not store any personal data.
+
+## Examples
+- Find all occurrences of "http://example.com/" followed by any number of digits on tables:
+
+    `php admin/tool/advancedreplace/cli/find.php --regex-match=http://example.com/\\d+`
```

Comment:
if you "" bash quote the params would this make the \\\\ just \ and read easier?

Reply:
I will fix this